Orange/preprocess/transformation.py

```diff
@@ -107,7 +107,10 @@ def __init__(self, variable, offset, factor):
         self.factor = factor
 
     def transform(self, c):
-        return (c - self.offset) * self.factor
+        if sp.issparse(c):
+            return c
```

Comment:
OK. You can check the current version and we can discuss later what to do with SVMs.